rpm/nfcd-binder-plugin.spec

```diff
@@ -17,6 +17,8 @@ BuildRequires: pkgconfig(nfcd-plugin) >= %{nfcd_version}
 Requires: libgbinder >= %{libgbinder_version}
 Requires: nfcd >= %{nfcd_version}
 
+%define plugin_dir /usr/lib/nfcd/plugins
```

Comment:
I guess it's better to rewrite the build script to support `/usr/lib64` as well.

Reply:
It's not just the build script, it's also the place where `nfcd` looks for plugins. There's no reason to have two nfcd's installed simultaneously, therefore there's no need for both 32-bit and 64-bit plugins installed simultaneously and no need for architecture specific plugins directories, so I decided to leave it as is for now.

I'm not strictly against 32-bit and 64-bit plugins sitting in different folders, it's just that it would require more changes than fixing 64-bit build like this. It's quite possible that it will happen at some point, for consistency's sake if nothing else.